tasks/managed/close-advisory-issues/close-advisory-issues.yaml

```diff
@@ -0,0 +1,100 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: close-advisory-issues
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >
+    Tekton task to close all issues referenced in the releaseNotes. It is meant to run after
+    the advisory is published. A comment will be added to each closed issue with a link
+    to the advisory it was fixed in
+  params:
+    - name: dataPath
+      description: Path to the JSON string of the merged data to use in the data workspace
+      type: string
+    - name: advisoryUrl
+      description: The url of the advisory the issues were fixed in. This is added in a comment on the issue
+      type: string
+  workspaces:
+    - name: data
+      description: The workspace where the snapshot spec json file resides
+  steps:
+    - name: close-issues
+      image: quay.io/konflux-ci/release-service-utils:0f82be4be43294b6a96846d87ef7f7c0b9e34267
+      env:
+        - name: ACCESS_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: todo
+              key: TODO
+      script: |
+        #!/usr/bin/env bash
+        # We do not set -x here because it would leak the ACCESS_TOKEN
```

Comment:
But it would be good to still have at least `set -eu` I'd say? (Especially since you already have it in mocks.sh, so now the test runs are not representative of what is actually run in production.)

Reply:
added